spec/support/database_cleaner.rb

```diff
@@ -0,0 +1,31 @@
+RSpec.configure do |config|
+  config.before(:suite) do
+    DatabaseCleaner.clean_with(:truncation)
+    Test.setup_models
+    Rails.application.load_seed
+  end
+
+  config.before(:each) do
+    DatabaseCleaner.strategy = :transaction
+  end
+
+  config.before(:each, js: true) do
+    DatabaseCleaner.strategy = :truncation
```

Comment:
Not only am I familiar with that project, but the code there came from this project! :)

https://github.com/18F/e-manifest/commit/c86e5d934d2d0bd47e0ec9fbedb67c29cbb0b1ec#diff-93830fa29d616f7c87903d08b5b1b29a